pkg/authorization/authorizer/scope/converter.go

```diff
@@ -130,43 +140,54 @@ func (clusterRoleEvaluator) Handles(scope string) bool {
 }
 
 func (e clusterRoleEvaluator) Validate(scope string) error {
-	_, _, err := e.getRoleNamespace(scope)
+	_, _, _, err := e.parseScope(scope)
 	return err
 }
 
-func (e clusterRoleEvaluator) getRoleNamespace(scope string) (string, string, error) {
+func (e clusterRoleEvaluator) parseScope(scope string) (string /*role name*/, string /*namespace*/, bool /*escalating*/, error) {
```

Comment:
If we're going to use this syntax, which I will always fight, we should at least put the comments in the place that Golang would put the names ...